Website/Website/auth.js

```diff
@@ -207,6 +212,8 @@ function initialize(host) {
         let url;
         if (env.localServices[query.service]) {
             url = `${env.localServices[query.service]}/api/${query.api}`;
+        } else if (kubernetesServices != null && `${kubernetesServices[query.service]}`) {
```

Comment:
access kubernetesServices as env.kubernetesServices. Also, in javascript, its common to use if (variable) to check for non-null and non-undefined. So in this example, you can simply the statement to be:
"} else if (env.kubernetesServices && env.kubernetesServices[query.services]) {"

Notice in my example above, I removed the `${env.kubernetesServices[query.service]}}`, you don't need it.

Reply:
updated